doc/nrf/libraries/bluetooth_services/services/cgms.rst

```diff
@@ -0,0 +1,24 @@
+.. _cgms_readme:
+
+GATT Continuous Glucose Monitoring Service (CGMS)
+#################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This module implements the Continuous Glucose Monitoring Service with the corresponding set of characteristics defined in the `Continuous Glucose Monitoring Service Specification`_.
```

Comment:
Well, we use this nomenclature in other service RST files (see BMS, HIDS). I guess we should stick to the module term so that it is aligned with other service docs